StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1625ElementDocumentationMustNotBeCopiedAndPasted.cs

```diff
@@ -75,7 +80,40 @@ internal class SA1625ElementDocumentationMustNotBeCopiedAndPasted : DiagnosticAn
         /// <inheritdoc/>
         public override void Initialize(AnalysisContext context)
         {
-            // TODO: Implement analysis
+            context.RegisterCompilationStartAction(HandleCompilationStart);
+        }
+
+        private static void HandleCompilationStart(CompilationStartAnalysisContext context)
+        {
+            context.RegisterSyntaxNodeActionHonorExclusions(HandleDocumentationTrivia, SyntaxKind.SingleLineDocumentationCommentTrivia);
```

Comment:
:bulb: This will need to be updated to match the cached delegate pattern used in all the other analyzers.

1. Cache the delegate in `DocumentationCommentTriviaAction`
1. Cache the immutable array containing both syntax kinds
1. Only call `RegisterSyntaxNodeActionHonorExclusions` once, and pass in the cached delegate and immutable array